BoxContentSDK/BoxContentSDK/Clients/BOXContentClient+File.h

```diff
@@ -170,6 +171,7 @@
  */
 - (BOXFileUploadNewVersionRequest *)fileUploadNewVersionRequestWithID:(NSString *)fileID
                                                     fromLocalFilePath:(NSString *)localFilePath;
+__attribute__((deprecated("Foreground fileuploads will be removed. Please use fileUploadNewVersionRequestInBackgroundWithFileID")));
```

Comment:
Anyone else, please comment on if there is a reason against disabling foregroup uploads